frontend/public/models/yaml-templates.ts

```diff
@@ -229,6 +233,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: example
+  annotations:
+    image.openshift.io/triggers: ${sampleContainerImage}
```

Comment:
If we're using `image.openshift.io/triggers`, we should refer to the image stream. This should look like

```suggestion
    image.openshift.io/triggers: |-
      [
        {
          "from": {
            "kind": "ImageStreamTag",
            "name": "openshift/hello-openshift:latest"
          },
          "fieldPath": "spec.template.spec.containers[0].image"
        }
      ]
```
```

Reply:
@spadgett Updated, thanks.